docs/04-guides/on-demand/01-upload-video-asset/02-api.mdx

```diff
@@ -208,3 +210,101 @@ curl --location --request GET 'https://livepeer.studio/api/asset/{id}' \
 
 </CodeBlockTabItem>
 </CodeBlockTabs>
+
+## Resumable Upload
+
+The resumable upload method allows a user to resume their upload if their 
+connection fails due to a cancelled request or a dropped connection. If you're
+uploading a large file and you lose connection at 98%, you don't have to restart
+from the beginning, you can continue back at 98%.
+
+### Step 1: Generate upload URL
+
+To upload the asset to the Livepeer network locally, you'll need to make a POST
+request to generate an upload URL.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="Node" label="Node">
```

Comment:
@ichung08 It's confusing that we tell users to use a mix of server and client side code. Let's be consistent across all code blocks. Either we assume everything is written on the client or the server.

Reply:
I think we initally decided on naming it as node but I've been writing these examples with client side implementation in mind. Do you want  them all renamed to js? But then we'd have to decide whether the optimal way of using livepeer api is through client or server side.

Or I can rewrite this code block to be called through server side.

What are your thoughts?

Reply:
Yeah let's rename them all to "JavaScript" and make it clear that these guides assume client side development 👍 We can note somewhere that developers can also use the API on the server side.

Reply:
I like that - will open a new ticket to address this change for existing guides

Reply:
Updated!